launch/world_modeling/people_recognition.launch

```diff
@@ -13,16 +13,44 @@
     <arg name="openface_srv_name" default="face_recognition/recognize" />
     <arg name="openpose_srv_name" default="/$(arg robot_name)/openpose/recognize" />
     <arg name="keras_srv_name" default="face_recognition/get_face_properties" />
+    <arg name="enable_topic_mode" default="False" />
+    <arg name="camera_info_depth" default="/$(arg robot_name)/head_rgbd_sensor/depth_registered/camera_info" />
+    <arg name="image_depth" default="/$(arg robot_name)/head_rgbd_sensor/depth_registered/image" />
+    <arg name="image_rgb" default="/$(arg robot_name)/head_rgbd_sensor/rgb/image_raw" />
 
     <arg name="openface_db" default="" />
     <arg name="openface_config" default="$(env ROBOT_BRINGUP_PATH)/parameters/world_modeling/face_recognition.yaml" />
     <arg name="keras_face_model" default="~/data/keras_models/age_gender/weights.28-3.73.hdf5" />
+    <arg name="openpose_models" default="~/openpose/models/" />
+
+    <group ns="openpose">
+        <!-- Openpose node -->
+        <node pkg="image_recognition_openpose" type="openpose_node" name="openpose_node" output="log" machine="$(arg machine)">
+            <param name="net_input_width" value="368" />
+            <param name="net_input_height" value="368" />
+            <param name="net_output_width" value="368" />
+            <param name="net_output_height" value="368" />
+            <param name="model_folder" value="$(arg openpose_models)" />
+            <remap from="image" to="openpose_input_image" />
+        </node>
+
+    </group>
```

Comment:
You should move the openpose namespace to the group with the real robot. Because openpose should only launch if the real robot is used and not when simulation is used.